concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1144,6 +1144,34 @@ jobs:
 
                   cf push
 
+        - task: sync-admin-users
+          config:
+            platform: linux
+            image: docker:///ruby#2.2
```

Comment:
Could we use [the cf-uaac container image](https://github.com/alphagov/paas-docker-cloudfoundry-tools/tree/master/cf-uaac)?
This would be faster, and its version of uaac is more recent.